webapp/channels/src/components/admin_console/custom_enable_disable_guest_accounts_setting.tsx

```diff
@@ -19,8 +19,16 @@ type Props = {
     showConfirm: boolean;
 }
 
-export default class CustomEnableDisableGuestAccountsSetting extends React.PureComponent<Props> {
-    public handleChange = (id: string, value: boolean, submit = false) => {
+const CustomEnableDisableGuestAccountsSetting: React.FC<Props> = ({
+    id,
+    value,
+    onChange,
+    cancelSubmit,
+    disabled,
+    setByEnv,
+    showConfirm,
+}) => {
+    const handleChange = useCallback((id: string, value: boolean, submit = false) => {
```

Comment:
Not sure if the linter will complain, but you are redefining here several variables (id and value) which makes things a bit more confusing. Do you mind renaming the variables?

I guess they could be `targetId`, and `newValue`, but I am horrible at naming...